### PR TITLE
[feature]

### PR DESCRIPTION
**Pull request by andres-mora-vanegas**
Added footer javascript file to handle all the logic in the same file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
 </head>
 <body>
     <h1 class="test">hola mundo</h1>
+    <footer>
+        sección de footer
```
Comment (YaiPV) [inline index.html]: I think this should not be here.

**Review by YaiPV — CHANGES_REQUESTED** [review]
Please check the comments.

File: index.html (hunk 1 above)
Comment (andres-mora-vanegas) [reply index.html]: Got it, the changes was done.

**Review by YaiPV — APPROVED** [review]
Approved

**Pull request merged**